### ISSUE TITLE
Special event "on top of" is ambiguous

### ISSUE THREAD

**Issue by djrenren** [andymeneely/project-timber-wolf#7]
During scenario setup instructions such as:
`https://github.com/andymeneely/project-timber-wolf/blame/a1e839e96b56df0532400b46c7b82b1fdc6eb221/scenarios/scenarios.md#L142

The instructions read "Place X on top of event 13" but the lack of instructions around how to stack the event cards makes this ambiguous. Consider "before" or "after" or even "between 13 and 14".

**Comment by andymeneely** [comment]
I believe I actually have some better instructions on the event cards that you never saw in v15. You'll be seeing those tonight in the new rules.

**Comment by andymeneely** [comment]
Yeah I'll fix this in the next one. "On top of 13" is intended to mean "between 12 and 13".
